Knowledge/react-app/src/components/HeroImage.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import Searchbar from './Searchbar';
+import '../css/heroImage.scss';
+
+class HeroImage extends Component {
+  render() {
+    return (
+      <div>
+        <div className="image"></div>
```

Comment:
have a naming convention like hero__img hero__title as image and title are classes that can be repeated through the project